src/components/modals/export-map-modal.js

```diff
@@ -58,41 +47,162 @@ const StyledInput = styled.input`
   }
 `;
 
-const propTypes = {
-  exportHtml: PropTypes.object,
-  onExportMapboxAccessToken: PropTypes.func.isRequired
+const DISCLAIMER = "* If you don't provide your own token, the map may fail to display at any time as we rotate ours to avoid misuse." +
```

Comment:
`when` we `invalidate` ours to avoid misuse (less technical sounding)

Reply:
done